### ISSUE TITLE
Unknown field "fsGroup"

### ISSUE THREAD

**Issue by lsarecz** [oracle/oci-service-broker#6]
When I try to install Service Broker and in Quick Setup I execute the command:
`helm install --devel charts/oci-service-broker/. --name oci-service-broker --set ociCredentials.secretName=ocicredentials --set storage.etcd.useEmbedded=true --set tls.enabled=false`
I get an error:
`Error: validation failed: error validating "": error validating data: ValidationError(Deployment.spec.template.spec.containers[0].securityContext): unknown field "fsGroup" in io.k8s.api.core.v1.SecurityContext`

Helm version:
```
Client: &version.Version{SemVer:"v2.14.0", GitCommit:"05811b84a3f93603dd6c2fcfe57944dfa7ab7fd0", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.14.0", GitCommit:"05811b84a3f93603dd6c2fcfe57944dfa7ab7fd0", GitTreeState:"clean"}
```

Based on [#5750](https://github.com/helm/helm/issues/5750) helm version v2.14.0 has a bug, which caused this issue, but I'm not sure.

**Comment by ashokkkannan** [comment]
Yeah, it is related with v2.14.0 issue. We will fix our manifest in the next release. If possible, till then can you please use v2.13.1

**Comment by lsarecz** [comment]
OK, how can I downgrade to v2.13.1? Or should I reinstall? Sorry for the dumb questions, I'm new with helm.

**Comment by ashokkkannan** [comment]
unfortunately, it is not possible to downgrade helm with brew easily. As a temporary workaround, you can download the helm binary directly and use it:

```
# Download older version of helm to tmp directory
cd /tmp && curl -SL https://storage.googleapis.com/kubernetes-helm/helm-v2.13.1-darwin-amd64.tar.gz | tar -zxf - darwin-amd64/helm 
export PATH=/tmp/darwin-amd64:$PATH

# Downgrade tiller version. After this command wait for 10-15 seconds for the upgrade to complete.
helm init --upgrade --wait --force-upgrade

# Verify version
helm version

# install chart using old version
helm install https://github.com/oracle/oci-service-broker/releases/download/v1.1.0/oci-service-broker-1.1.0.tgz  --name oci-service-broker  --set ociCredentials.secretName=ocicredentials  --set storage.etcd.useEmbedded=true  --set tls.enabled=false
```

**Comment by ashokkkannan** [comment]
@lsarecz This issue is fixed in the latest release [v1.1.1](https://github.com/oracle/oci-service-broker/releases/tag/v1.1.1). Please let us know if you still  having this issue with the new version.

**Comment by ashokkkannan** [comment]
We are closing this issue as it is fixed. Please reopen if you are still seeing this issue.
